<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>well_manage_Dialog</class>
 <widget class="QDialog" name="well_manage_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Well Management</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icon/data_icon</normaloff>:/icon/data_icon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="wells_listWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QPushButton" name="rename_well_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Rename Well</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/rename_icon</normaloff>:/icon/rename_icon</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="delete_well_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Delete Well</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/delete_icon</normaloff>:/icon/delete_icon</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="import_well_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Import Well</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/import_icon</normaloff>:/icon/import_icon</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="layer_well_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Well Layer Markers</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/layer_icon</normaloff>:/icon/layer_icon</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="export_well_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Export Well</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/export</normaloff>:/icon/export</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="edit_well_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Edit Well Info</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/edit_icon</normaloff>:/icon/edit_icon</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="logs_listWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QPushButton" name="view_log_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>View Log curve</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/view_icon</normaloff>:/icon/view_icon</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="delete_log_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Delete Log</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/delete_icon</normaloff>:/icon/delete_icon</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="rename_log_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Rename Log</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/rename_icon</normaloff>:/icon/rename_icon</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="import_log_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Import Log</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/import_icon</normaloff>:/icon/import_icon</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="edit_log_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Edit Log data</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/edit_icon</normaloff>:/icon/edit_icon</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="create_log_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>190</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Create new Log curve</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/create_icon</normaloff>:/icon/create_icon</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="export_log_Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>25</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Export Log</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icon/export</normaloff>:/icon/export</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTextEdit" name="info_textEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="saved_info_tableWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>keys</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>value</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
